bdai_ros2_wrappers/bdai_ros2_wrappers/subscription.py

```diff
@@ -78,3 +81,137 @@ def wait_for_message(
         future.cancel()
         return None
     return future.result()
+
+
+def wait_for_messages(
+    node: rclpy.node.Node,
```

Comment:
@kaiyu-zheng nit: consider making `node` optional, fetching it from the outer `scope` by default like the other functions in this module do.

Reply:
Done.